source/agora/node/FullNode.d

```diff
@@ -275,27 +275,27 @@ public class FullNode : API
         {
             // Make `BlockExternalizedHandler`s from config
             config.event_handlers.filter!(h => h.handler_type == HandlerType.block_externalized)
-                .map!(handler => handler.handler_addresses).map!(a => a.to!Address)
-                    .each!(address =>
-                        this.block_handlers[address] = this.network.getBlockExternalizedHandler(address));
+                .each!(handler => handler.handler_addresses
+                    .each!((string address) =>
```

Comment:
This is not necessary though, we used it just to debug. But I don't mind if you want to keep it.

Reply:
> did `map!(a => a.to!Address)` imply a `string` type?

It did but the problem was a misplaced bracket which meant it was mapping a list to a list (which became "[item1, item2]" when converted to a `string` so the address was corrupted.

Reply:
I'll keep it just to remind people that the type can be added (which helps with debugging when compile fails).